apps/brainstorm/.dockerignore

```diff
@@ -1,3 +0,0 @@
-*
-!config
-!scripts
```

Comment:
Leave it with `*` and `!code` then.

Reply:
`.dockerignore` is there to prevent anything unwanted to reach the image. Keep this file.

Reply:
Will do